### PR TITLE
Feature/component modals

### PR DESCRIPTION
**Pull request by SyleimanovSergey**

### CODE REVIEW

File: src/pages/MainPage/index.tsx [hunk 1]
```diff
@@ -10,18 +10,24 @@ import { getMembers } from 'redux/reducer/selectors';
 import { btn } from 'constants/commonComponents';
 import MyButton from 'components/common/MyButton';
 import MyInput from 'components/common/MyInput';
-import FormRegistration from 'components/Forms/RegistrationForm';
 
 import pokerPlanningImage from 'assets/images/MainPage/poker-planning.jpg';
 
 import styles from './index.module.scss';
+import Modals from '../../components/common/Modals';
+import { TypeModalsOpen } from '../../interfaces/modals';
```
Comment (Polina04) [inline src/pages/MainPage/index.tsx]: `../../.../../../../../../../../`

Comment (Polina04) [reply src/pages/MainPage/index.tsx]: please fix imports in this file

File: src/pages/MainPage/index.tsx [hunk 2]
```diff
@@ -63,12 +69,8 @@ const MainPage: React.FC = (): JSX.Element => {
...
+				{openModal && <Modals open={setOpenModal} type={TypeModalsOpen.kick} />}
 			</div>
-			{form && (
-				<div className={styles.form}>
-					<FormRegistration setForm={setForm} />
-				</div>
-			)}
 		</div>
 	);
 };
```
Comment (Polina04) [inline src/pages/MainPage/index.tsx:73]: `setAdmin` unused function
do you really need it?

Comment (Polina04) [reply src/pages/MainPage/index.tsx:73]: please add empty lines before returns in functions `setAdmin` and `setUser`

File: src/components/ModalKick/index.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import MyButton from 'components/common/MyButton';
+import { btn } from 'constants/commonComponents';
+
```
Comment (Polina04) [inline src/components/ModalKick/index.tsx]: redundant empty line

File: src/components/ModalKick/index.tsx [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+				<p className={styles.text}>
+					Are you really want to remove player
+					<span className={styles.player}>` ${player} `</span>
+					from game session?
+				</p>
+			</div>
+		);
+	};
+
+	const TypeSubmitBtn = () => {
```
Comment (Polina04) [inline src/components/ModalKick/index.tsx]: please rename it to `onSubmit` or something like this

File: src/components/ModalKick/index.tsx [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+		return vote ? console.log('vote') : console.log('admin');
+	};
+
+	return (
+		<div className={styles.kickModal}>
+			<TypeKickModal />
```
Comment (Polina04) [inline src/components/ModalKick/index.tsx]: please use ternary operator instead of `<TypeKickModal />`

**Review by Polina04 — COMMENTED** [review]
something went wrong with prettier(

**Review by DmitriyHnatiuk — APPROVED** [review]

**Review by NiFroPP — APPROVED** [review]

**Pull request merged**